lib/paperclip/storage/fog.rb

```diff
@@ -217,14 +217,16 @@ def connection
         @connection ||= ::Fog::Storage.new(fog_credentials)
       end
 
-      def directory
-        dir = if @options[:fog_directory].respond_to?(:call)
+      def directory_name
```

Comment:
I'd define this after `directory`, as it's used by it and file will read better.